Chinese/诫子书/君子.md

```diff
@@ -0,0 +1,2 @@
+夫君子之行，静以修身，俭以养德，非淡泊无以明志，非宁静无以致远
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.